dvc/command/experiments.py

```diff
@@ -1249,6 +1250,12 @@ def add_parser(subparsers, parent_parser):
         action="store_true",
         help="Remove all committed experiments.",
     )
+    experiments_remove_parser.add_argument(
```

Comment:
Should this be part of the mutually exclusive group:

```suggestion
    remove_group.add_argument(
```

Using `-g` with `--queue` doesn't make sense. `dvc exp remove --all -g` doesn't seem to do anything, and I think we discussed that `--all` is probably not wanted for now either since it may be dangerous/unexpected in a collaborative repo.

Reply:
If we will not have any of these kinds of feature requests.